### ISSUE TITLE
Whitespace in path name causes error

### ISSUE THREAD

**Issue by VinnieMuffinMan** [34j/vscode-black#24]
## Current Behavior

When running the extension with a path name that has a whitespace in it, the command doesn't put quotes around the path and so the path isn't valid.

## Expected Behavior

The extension should assume that there are whitespaces and preemptively add quotes to all path names.

## Steps to Reproduce the Problem

  1. Run the extension on a file whose path has a space.
  2. Black raises an Invalid value for 'SRC ...' error.

## Environment

- Version: v1.0.3
- Platform: Windows
- Node.js Version: v18.18.0

**Comment by Lemonmous** [comment]
有解决方法吗？

**Comment by Davo36** [comment]
Yeah I just installed this and found the same error.  I have my code on Google Drive.  And the path includes "My Drive" so it fails.  Oh well, uninstalling and trying another one.
